.github/workflows/go-test.yaml

```diff
@@ -112,6 +112,22 @@ jobs:
           kubectl -n rook-ceph scale deployment rook-ceph-osd-0 --replicas 0
           kubectl rook-ceph rook purge-osd 0 --force
 
+      - name: Restore CRD without CRName
+        run: |
+          # First let's delete the cephCluster
+          kubectl -n rook-ceph delete cephcluster my-cluster --timeout 3s || true
+
+          kubectl rook-ceph -n rook-ceph restore cephcluster
+          tests/github-action-helper.sh wait_for_crd_to_be_ready_default
```

Comment:
What state is the CR in during this action? Will it require a successful reconcile to get back to Ready state? Looks like the CI is taking over a minute to wait for the Ready state. That seems longer than I would expect, but maybe that's normal.

Reply:
in my local testing, local I have seen it takes around a minute or full reconciliation to mark this CR ready at least for the cephCluster cr

Reply:
I'll test again and see

Reply:
```
~/go/src/github.com/kubectl-rook-ceph
srai@192 ~ (restore-crd) $ kc get cephcluster
NAME         DATADIRHOSTPATH   MONCOUNT   AGE   PHASE         MESSAGE                   HEALTH   EXTERNAL   FSID
my-cluster   /var/lib/rook     1          61s   Progressing   Configuring Ceph Mgr(s)                       
~/go/src/github.com/kubectl-rook-ceph
srai@192 ~ (restore-crd) $ kc get cephcluster
NAME         DATADIRHOSTPATH   MONCOUNT   AGE   PHASE   MESSAGE                        HEALTH        EXTERNAL   FSID
my-cluster   /var/lib/rook     1          64s   Ready   Cluster created successfully   HEALTH_WARN              5dc8aca4-cb12-47ee-a83e-a9f773b39a0a
```

it took around 1 min